ci-operator/step-registry/interop-tests/cnv-tests-e2e-deploy/interop-tests-cnv-tests-e2e-deploy-commands.sh

```diff
@@ -4,6 +4,29 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function check_if_yq_exists() {
+    # Install yq manually if its not found in image
+    echo "Checking if yq exists"
+    cmd_yq="$(yq --version 2>/dev/null || true)"
+    if [ -x "${cmd_yq}" ]; then
+        echo "Installing yq"
+        curl -L "https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/')" \
+         -o ./yq && chmod +x ./yq
+    fi
+}
+
+
+function mapTestsForComponentReadiness() {
+    if [[ $MAP_TESTS == "true" ]]; then
+        results_file="${ARTIFACT_DIR}/junit.functest.xml"
+        if [ -f $results_file ]; then
+            check_if_yq_exists
+            echo "Mapping Test Suite Name To: CNV-lp-interop"
+            yq eval -px -ox -iI0 '.testsuites.testsuite.+@name="CNV-lp-interop"' $results_file
+        fi
```

Comment:
I would extract this to a function, and call it with the file as argument.

```suggestion
        patch_junit_file "${ARTIFACT_DIR}/junit.functest.xml"
```